### ISSUE TITLE
Chapter code and video are mismatched

### ISSUE THREAD

**Issue by syobochim** [PacktPublishing/React---The-Complete-Guide-includes-Hooks-React-Router-and-Redux-Second-Edition#5]
The code for Chapter 22 and Chapter 23 are swapped. The Deployment Steps code is included in Chapter 22, while Adding Authentication to React Apps is included in Chapter 23. However, the order is reversed in the video.

Links:
https://github.com/PacktPublishing/React---The-Complete-Guide-includes-Hooks-React-Router-and-Redux-Second-Edition/tree/main/Section%2022
https://github.com/PacktPublishing/React---The-Complete-Guide-includes-Hooks-React-Router-and-Redux-Second-Edition/tree/main/Section%2023

**Comment by syobochim** [comment]
There is a mismatch between the code in Chapter 21 on GitHub and the code being explained in Chapter 22 "Deploying React Apps" video. The video features the Blog Post service, but the code features the Great Quotes service instead.

<img width="770" alt="image" src="https://user-images.githubusercontent.com/3622660/235581802-9dbffd09-4699-4ae3-bd9d-d93e19a5be12.png">

**Comment by Sonakshis** [comment]
Hi,

We had a look at the files. The files related to "Section 21: Authentication" are part of the Section 21 folder, and files related to "Section 22: Deploying React Apps" are available in the Section 22 folder.

We could not find any extra or incorrect files in the Section 23 folder; all the resources are for the video "An Introduction to Next.js"

For easy understanding, we have added the course ToC to the readme file with section numbers and names. Also, we have updated the section 21 and 22 folders with the section names.

Happy Learning!
